### ISSUE TITLE
Flipp Weekly Flyers App & Flipp website getting blocked Adguard DNS on router

### ISSUE THREAD

**Issue by ziomeh** [AdguardTeam/AdGuardSDNSFilter#97]
I'm using Adguard DNS servers ( 176.103.130.130 and 176.103.130.131 ) on my home network in Canada.

Everything was working fine up until June 2019 but off late it is giving trouble with legit Android apps and websites like Flipp Weekly Flyers and their corporate website https://www.flipp.com.

I get **_This site can’t be reached www.flipp.com’s server IP address could not be found._** error on trying to access this site.

What is the issue here? I have checked everything else on my end. The site promptly opens up and the Android app also loads all flyers with thumbnails when I use 4G/LTE on my phone with cellular data!

Please look into it and rectify the error. 90% of Canadians use the Flipp app regularly and all flyers are getting blocked!

Thanks!

**Comment by Alex-302** [comment]
This is caused by the rule from EasyList. Excluded from DNS filter:
https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/98

**Comment by WildByDesign** [comment]
To see the image thumbnails/pictures in the Flipp app, you also need to whitelist:

`f.wishabi*`


Here is what I had to add to my domain whitelist:

```
flipp.com
f.wishabi*
wishabi.net
f.wishabi.net
```


See: https://flipp.force.com/articles/en_US/I-can-t-see-images-of-my-weekly-ads-flyers

**Comment by Alex-302** [comment]
I don't see requests to `wishabi.net`

**Comment by WildByDesign** [comment]
Here is some of the domains queried from Flipp app this morning in Ontario, Canada:

```
images.wishabi.net
f.wishabi.net
b.wishabi.com
a.wishabi.com
avro-beacon-production.wishabi.com
```

From my own experience, I whitelisted the **flipp.com** domain about a year ago. Just within the last 2-3 weeks, the app continued to work, however, all images in the app were broken. So once I whitelisted the **wishabi.net** domain, that fixed the issue with no images showing in the app.

But to be more specific and less broad with the full **wishabi.net** domain, likely just need to add:

```
images.wishabi.net
f.wishabi.net
```
